package.json

```diff
@@ -17,7 +17,7 @@
     "link:yarn": "lerna run --stream --concurrency 1 link:yarn",
     "lint": "lerna run --parallel lint",
     "lint:eslint": "lerna run --parallel lint:eslint",
-    "pack:changed": "lerna run pack --since",
+    "createTarball:changed": "lerna run createTarball --since",
```

Comment:
Will this `--since` flag still make sense being passed directly to the command once the build-reorg changes go through?

Reply:
According to the [Lerna Readme](https://github.com/lerna/lerna/tree/main/core/filter-options#--since-ref), the `--since` flag is used to only include packages that were changed since the last tag (the last tag is used as a default ref if nothing else is passed). So in this command, it should only pack tarballs that changed since the last tag (??). I'm not entirely sure though, if this is in fact working or giving us any kind of advantage here.

Reply:
Yeah, in light of that I think we should just yank this entirely.

Reply:
Sounds good, will do

Reply:
I've added the `--parallel` flag as it seems to me that this is a task that doesn't have to be executed linearly. It won't give us a huge performance win as packing generally seems to only take a few seconds. On my machine, it reduces packing time by around 35% (which translates to a massive win of 5 seconds ;) ).